internal/pkg/config/v1/config.go

```diff
@@ -62,7 +62,11 @@ var (
 	)
 	RequireOnPremLoginErr = errors.NewErrorWithSuggestions(
 		"you must log in to Confluent Platform to use this command",
-		"Log in with `confluent login --url <mds-url>`.",
+		"Log in to Confluent Platform with `confluent login --url <mds-url>`.",
+	)
+	CloudLoginRequireOnPremLoginErr = errors.NewErrorWithSuggestions(
+		"this is not a Confluent Cloud command. You must log in to Confluent Platform to use this command",
```

Comment:
The second half of this error is the same as `RequireOnPremLoginErr`; is the first half here in case the user doesn't realize that they're logged into cloud?

Reply:
yes. they're logged into cloud but tried running a cp cmd. i'm unhappy with my naming. do you have suggestions?

Reply:
How about `RunningOnPremCommandInCloudErr`?

Reply:
Alternatively, you could just get rid of this one and just reuse `RequireOnPremLoginErr`. Hopefully even without the note about Cloud, that error should cause people to double check their login.

Reply:
it's getting used in a few places so easier to name this a separate variable